framework/include/wifi_manager/wifi_manager.h

```diff
@@ -322,136 +328,148 @@ wifi_manager_result_e wifi_manager_unregister_cb(wifi_manager_cb_s *wmcb);
  * @return On success, WIFI_MANAGER_SUCCESS (i.e., 0) is returned. On failure, non-zero value is returned.
+ * @API type: synchronous
+ * @callback: none
  * @since TizenRT v1.1
  */
-// API type: synchronous
-// callback: none
 wifi_manager_result_e wifi_manager_set_mode(wifi_manager_mode_e mode, wifi_manager_softap_config_s *config);
 
 /**
  * @brief Retrieve current status of Wi-Fi interface including mode, connection status, ssid, received signal strengh indication, and ip address.
  * @details @b #include <wifi_manager/wifi_manager.h>
  * @param[out] retrieved information including  mode, connection status, ssid, received signal strengh indication, and ip address.
  * @return On success, WIFI_MANAGER_SUCCESS (i.e., 0) is returned. On failure, non-zero value is returned.
+ * @API type: synchronous
+ * @callback: none
  * @since TizenRT v1.1
  */
-// API type: synchronous
-// callback: none
 wifi_manager_result_e wifi_manager_get_info(wifi_manager_info_s *info);
 
 /**
  * @brief Connect to an access point.
  * @details @b #include <wifi_manager/wifi_manager.h>
  * @param[in] config ssid, passphrase, authentication type, and crypto type of the access point which the wi-fi interface connect to.
  * @return On success, WIFI_MANAGER_SUCCESS (i.e., 0) is returned. On failure, non-zero value is returned.
+ * @API type: asynchronous
+ * @callback: sta_connected
  * @since TizenRT v1.1
  */
-// API type: asynchronous
-// callback: sta_connected
 wifi_manager_result_e wifi_manager_connect_ap(wifi_manager_ap_config_s *config);
 
-
 /**
  * @brief Disconnect from the connected access point
  * @details @b #include <wifi_manager/wifi_manager.h>
  * @param[in] none
  * @return On success, WIFI_MANAGER_SUCCESS (i.e., 0) is returned. On failure, non-zero value is returned.
+ * @API type: asynchronous
+ * @callback: sta_disconnected
  * @since TizenRT v1.1
  */
-// API type: asynchronous
-// callback: sta_disconnected
 wifi_manager_result_e wifi_manager_disconnect_ap(void);
 
 /**
  * @brief Scan nearby access points
  * @details @b #include <wifi_manager/wifi_manager.h>
  * @param[in] none
  * @return On success, WIFI_MANAGER_SUCCESS (i.e., 0) is returned. On failure, non-zero value is returned.
+ * @API_type: asynchronous
+ * @callback: scan_ap_done
  * @since TizenRT v1.1
  */
-// API type: asynchronous
-// callback: scan_ap_done
-wifi_manager_result_e wifi_manager_scan_ap(void);
+/* SCAN type
+ *     1) full scan: config is null
+ *     2) scan with specific SSID: ssid is set in config
+ *     3) scan with specific channel: channel is set in config
+ *     4) scan with specific SSID and channel: Both ssid and channel are set in config
+ *
+ * Notes
+ *     if both SSID and channel are not set in config then it'll return error.
+ *     wifi_manager check whether SSID is set by ssid_length in config. In the same way
+ *     wifi_manager check whether channel is set by checking channel is 0.
```

Comment:
Maybe not checking channel is 0, but checking channel is greater than 0 and less than 15 in 2.5GHz.